okapi/core/CronJob/Cron5Job.php

```diff
@@ -25,6 +25,7 @@
     public function get_next_scheduled_run($previously_scheduled_run)
     {
         $t = time() + $this->get_period();
-        return ($t - ($t % 300));
+
+        return $t - ($t % 300);
     }
```

Comment:
I agree in this case, but generally I would prefer to allow some bracketing of assigned terms. E.g. for terms containing `==` and for `? :`.